public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -416,6 +424,13 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
     : atomicAnomalyResults;
   const handleDateRangeChange = useCallback(
     (startDate: number, endDate: number, dateRangeOption?: string) => {
+      if (
+        !props.isHistorical &&
+        startDate < get(props, 'detector.enabledTime') &&
```

Comment:
If detector started a week ago, then user restart it, the `enabledTime` will change to now. If user choose to review last 24 hours, will we show out of range modal?

Reply:
Yes. Had a discussion with UX about this awhile back. It was agreed this was acceptable. A user may likely stop and restart a detector when making some configuration change - in this case, selecting any time prior to the latest enabled time may be desired to run historical analysis with the latest config changes.